app/scripts/modules/core/delivery/status/executionStatus.controller.js

```diff
@@ -50,4 +50,10 @@ module.exports = angular.module('spinnaker.core.delivery.executionStatus.control
       return 'Unknown';
     };
 
+    if ($scope.execution.trigger && $scope.execution.trigger.parameters) {
+      $scope.parameters = Object.keys($scope.execution.trigger.parameters).sort().map((paramKey) => {
+        return { key: paramKey, value: $scope.execution.trigger.parameters[paramKey] };
```

Comment:
Is there a reason that 'parameters' is being added to the $scope and not to the 'controllerAs' name?